dydx/dydxPresenters/dydxPresenters/_Features/routing_swiftui.json

```diff
@@ -303,6 +303,10 @@
                     "destination":"dydxPresenters.dydxClosePositionInputViewBuilder",
                     "presentation":"half"
                 },
+                "/trade/take_proft_stop_loss":{
```

Comment:
@mike-dydx Noticed there is a typo in "proft", when I copied the code to Android.